Gems/Atom/RPI/Code/3rdParty/FetchSPD.cmake

```diff
@@ -0,0 +1,34 @@
+# This script encapsulates fetching SPD as a dependency and providing the
```

Comment:
I added @o3de/sig-core-reviewers to this PR because I would like them to verify that it is acceptable to download a 3rdParty tool in this way.

Reply:
This file is missing a copyright header

Reply:
Instead of `FetchSPD.cmake`, perhaps in this case it's better just to incorporate the spd headers directly in source (in the shader asset folder under `External/FidelityFX` potentially), preserving the MIT copyright notice at the top, since this isn't a dependency that requires compilation or actual integration with the build system. Curious if @lumberyard-employee-dm or @lawsonamzn has a preference on this.

Reply:
Thanks.  I deleted FetchSPD.cmake and placed the SPD header files in the change.